nixos/modules/virtualisation/lxd.nix

```diff
@@ -177,7 +177,6 @@ in {
       "fs.inotify.max_queued_events" = 1048576;
       "fs.inotify.max_user_instances" = 1048576;
       "fs.inotify.max_user_watches" = 1048576;
-      "vm.max_map_count" = 262144;
```

Comment:
turn this into an assert to test for ≥ 262144

Reply:
This is set only if `virtualization.lxd.recommendedSysctlSettings` is set to `true` (and it's `false` by default).